src/wp-content/themes/twentynineteen/sass/site/primary/_comments.scss

```diff
@@ -318,7 +318,8 @@
 
 		a {
 			text-decoration: underline;
-
```

Comment:
Let's keep this empty line for consistency with other SASS rules :)

Reply:
My bad, sorry! Thank you for pointing this out.